app/views/attachment_mailer/new_attachment_notification.html.haml

```diff
@@ -1,23 +1,40 @@
-%table.full-width.header
-  %tr
-    %td.hero
-    %td
-      = image_tag "emails/icon-pencil-circle.png", class: "header-icon"
+- icon = "emails/icon-clippedPage.png"
```

Comment:
cool I like ths strategy of naming the dynamic vars at the top. not common in view partials but will probably save us time here!